### ISSUE TITLE
Docker needed

### ISSUE THREAD

**Issue by taofuyu** [Deci-AI/super-gradients#1124]
### 💡 Your Question

Although, the README says `pip install super-gradients` is all needed, but it reports many errors when I run some code based on super-gradients in fact.
I use pytorch docker from https://catalog.ngc.nvidia.com/containers, for example:
`docker pull nvcr.io/nvidia/pytorch:22.07-py3`

And in this container: 
`pip install super-gradients`

Then I run a dummy script:
`from super_gradients.training import models`
`from super_gradients.common.object_names import Models`
`print('hello')`

It follows so many environment errors.

If you can provide a dockerfile or docker image, it will help a lot, thanks.
